src/codegen/debug.rs

```diff
@@ -53,6 +57,27 @@ impl From<DebugLevel> for DWARFEmissionKind {
 /// An implementor of this trais will be called during various codegen phases to generate debug
 /// information
 pub trait Debug<'ink> {
+    /// Set the debug info source location of the instruction currently pointed at by the builder
+    fn set_debug_location(
+        &self,
+        llvm: &Llvm,
+        scope: &FunctionValue,
+        //Current line starts with 0
+        line: usize,
+        column: usize,
+    ) -> Result<(), Diagnostic>;
+
+    /// ...
```

Comment:
give me half a sentence

Reply:
This slipped throuch, I thought I removed these :D